### PR TITLE
Add to deliverables file

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR adds | fixes:

- Adds output files to store for
  - gatk_combinevariants recipe
  - sambamba depth recipe
  - chromograph recipe
  - Use MIPs bcftools singularity image for processing VT references

### How to test:
1. Automatic and continuous test suite
#### Deliverables
1. Run `cg workflow mip-dna run -p low  vitalmouse`
#### Bcftools binary
1. Run `cg workflow mip-dna run -p low  vitalmouse`

### Expected outcome:
1. Installation, unit and integration test suite pass
1. gatk_combinevariants, sambamba depth and chromograph recipe outfiles should be in the deliverables file ✅ 
1. Reference are check for VT processing ✅ 

### Review:
- [ ] Code review
- [ ] Tests pass approved by @henrikstranneheim 

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### How to test:
1. Run `cg workflow mip-dna run -p low  vitalmouse`

### Expected outcome:
1. gatk_combinevariants, sambamba depth and chromograph recipe outfiles should be in the deliverables file ✅ 

Screenshot 2020-04-28 at 20 46 04

Screenshot 2020-04-28 at 20 46 40

Screenshot 2020-04-28 at 20 47 04

**Comment by henrikstranneheim** [comment]
> Looks good!
> Isn't this a minor though, we're adding information to one of our main outfiles?

Yes, but there is no new functionality code wise. Just addition of data that was supposed to be there.

**Comment by jemten** [comment]
ready for new review?

**Comment by henrikstranneheim** [comment]
That would be great!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,23 @@
...
+- Chromograph now creates ideograms over the chromosomes
+- Proband chromograph files are named with sample id instead of case id
+- Increases java memory for picardtools markduplicates
+
+## [8.2.2]
+- Fixes a crash in MIP caused by not looping over the Y chromosomes for females in the GATK SplitNCigarReads recipe
+
+## [8.2.1]
+- Removes multiple identical entries in the store file. Keeps the most recent
+
```
Comment (jemten) [inline CHANGELOG.md]: Good additions!

**Comment by henrikstranneheim** [comment]
### How to test:
#### Bcftools binary
1. Run `cg workflow mip-dna run -p low  vitalmouse`

### Expected outcome:
1. Reference are check for VT processing ✅ 

Screenshot 2020-05-06 at 21 05 58

**Comment by henrikstranneheim** [comment]
@jemten Could you have a another look, please - have now added the bcftools binary using singularity img?

**Pull request merged**